### DISCUSSION TITLE
Is a new release planned?

### DISCUSSION THREAD

**Discussion by Bubbi** [jbaysolutions/vue-grid-layout#626] [Q&A]
I think this is one of the best dashboard grid layouts for vue.
I noticed that it doesn't support **transform: scale();** all that well, but there are some pull requests for it. It could be nice to get implemented.

My usecase is that I scale down the grid layout to 75% while in editing mode, so the user can drag and drop the items around easier as well as add new items from an external list on the side.

Is this planned? Or has the development stalled entirely on this great grid system?

**Comment by gmsa** [comment]
There is a release planned, but since the lib is in a state where is it relatively stable and is not being actively developed, things will go slower.

**Comment by shawnpyle** [comment]
Some beta versions have been made and pushed to npm.
https://www.npmjs.com/package/vue-grid-layout

**Comment by namespace-github** [comment]
@gmsa 
The **3.0.0-beta1** was released a year ago. I have been using this libary for many months with vue3 and have no problems - it runs stable.

Is an official 3.0 version in planning? The repository is still very popular. Thank you very much for your work.

**Reply by ulissepress** [reply]
same for me. When you will release an "official" version supporting Vue 3 (i..e 3.0.0-beta1 to become 3.0.0)
